app/editor/src/features/content/form/ClipDirectoryTable.tsx

```diff
@@ -0,0 +1,52 @@
+import { useFormikContext } from 'formik';
+import { IItemModel } from 'hooks/api-editor';
+import React from 'react';
+import { GridTable } from 'tno-core';
+
+import { ClipDirectoryFilter } from './ClipDirectoryFilter';
+import { clipDirectoryColumns } from './constants';
+import * as styled from './styled';
+
+export interface IClipDirectoryTableProps {
+  /** the data to be displayed in the table */
+  data: IItemModel[];
+  deleteItem: (item: IItemModel) => void;
+  selectItem: (item: IItemModel) => void;
+  downloadItem: (item: IItemModel) => void;
+  attachItem: (item: IItemModel) => void;
+  navigate: (item: IItemModel) => void;
+}
+
+/** Table used to display time log for users creating and updating content. */
+export const ClipDirectoryTable: React.FC<IClipDirectoryTableProps> = ({
+  data,
+  deleteItem,
+  selectItem,
+  downloadItem,
+  attachItem,
+  navigate,
+}) => {
+  const parsedData = data.map((d: IItemModel) => ({
```

Comment:
What's the difference between a `parsedData` and `data`?

Reply:
Again, I used some of Jonny's code as a template for this without understanding all the ins and outs. I'll give him a quick ping when he's back on Teams (he's not responding at the moment).

Reply:
Worked out how to remove the reference to parsedData. Passing "data" to the GridTable component worked fine.